client/package.json

```diff
@@ -3,7 +3,8 @@
     "version": "0.1.0",
     "private": true,
     "dependencies": {
-        "@react/react-spectrum": "^2.28.0",
+        "@react/collection-view": "file:react-collection-view-4.1.5.tgz",
+        "@react/react-spectrum": "file:react-react-spectrum-2.25.0.tgz",
```

Comment:
Where do we put these files? They are currently not in the PR afaics, so a build would fail to find them.

- Are we allowed to include them in our public git repo? Or was Simon's message assuming a private git.corp repo?
- And how large are they? (Trying to avoid large binaries in git)

Depending on answers:
- If we can include in the git repo: put into a subdirectory, say `client/libraries` (?).
- If not, we would have to accept the fact that we can't have a full self-contained open source build (i.e. `npm install` on Travis CI can't work, and we have to rely on our internal CI). We could publish these modules on our internal npm registry instead and reference there (e.g. `@nui/react-react-spectrum` unless they already exist somewhere).

Reply:
in the example simon sent, an **open sourced public repo on github.com** contains these files:
https://github.com/adobe/reactor-extension-core/blob/master/react-react-spectrum-2.25.0.tgz
https://github.com/adobe/reactor-extension-core/blob/master/react-collection-view-4.1.5.tgz


btw, the files in my PR were ignored by .gitignore and that it why I mistakenly forgot to commit them. They are there now

Reply:
Btw, I copied the same set up as the example repo Simon sent: https://github.com/adobe/reactor-extension-core

I will move them to a `client/libraries` folder right now

Reply:
Size seems to be ~1.2 MB. That's ok to add.